src/platforms/javascript/index.mdx

```diff
@@ -23,119 +23,57 @@ Using a framework? Take a look at our specific guides to get started.
 
 ## Install the Package
```

Comment:
```suggestion
## Install
```

Just `Install`?

Then we have 
`Install`
`Initialize`
`Test`

as headlines

Reply:
Sure - onboarding is using more words, but I like the symmetry of verb, verb, verb.